web/src/components/common.js

```diff
@@ -5,6 +5,25 @@ import Col from 'react-bootstrap/Col'
 import Card from 'react-bootstrap/Card'
 
 
+function change_url(x, name=null) {
+  if (!name) {
```

Comment:
Could you move the JS part in another PR ?

Reply:
I split it in 2 commits